### ISSUE TITLE
BUG - sev-3 - iOS - Claims file request photo screen almost unusable at sufficiently large font sizes

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#5034]
## What happened?

Claims photo upload used to look something like this at close-to-maximum font size on iOS (see ticket #2742):
![image.png](https://images.zenhubusercontent.com/61a5397dac83844bf079e942/8a793737-bc36-4b4b-b391-46fa62cfdcad)

Now it looks like this (I'm pretty sure these are different testing devices, but you get the idea):
![image.png](https://images.zenhubusercontent.com/61a5397dac83844bf079e942/7cb987f0-cc10-485d-8564-667111182308)

## Specs:

- Device:
- OS:
- User Account: arfan's Aug 19th claim should have an open file request. DO NOT FILL IT.
- Accessibility State: Accessibility sizes (for larger font) turned on, and used

## Desired behavior

I think we need to set a reasonable maximum for the icons in the claims photo screen. 

## Acceptance Criteria

**Option 1 & preference:** We keep the 3-column layout, but set max sizes for:
- [ ] Thumbnail images
- [ ] Icons within the thumbnails (set max size or to a percentage within the thumbnail)
- [ ] The thumbnail label (file size and/or "add photo" copy)

**Option 2:**
- [ ] We figure out a way to allow each of these elements to expand (similar to grid / flexbox) and wrap to help accommodate the increased font size.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Issue Tracking](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va-mobile-app/testing/VA%20Mobile%20App%20Test%20Plan.md#issue-tracking) for details on Severity Levels

- 3 - Low

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, feature, bug)
- [ ] Estimate of 1 added

**Comment by bischoffa** [comment]
Ticket is blocked by UX review for desired behavior. That other ticket will need to be prioritized and work before this can be considered for prioritization and work. No ETA on when the blocked ticket will be worked.

**Comment by bischoffa** [comment]
No longer blocked by UX as their work as been completed. Ready to be prioritized for future work

**Comment by DJUltraTom** [comment]
verified the fonts and buttons are looking more like the old style screen and not the problematic one
QA approves this change
